### ISSUE TITLE
[BUG] Paper trading mode doesn't start: TypeError: object NoneType can't be used in 'await' expression 

### ISSUE THREAD

**Issue by albiDmtr** [CoinAlpha/hummingbot#1570]
**Describe the bug**
I installed the bot from source, and when I start the bot in paper trading mode I get the following error:
Traceback (most recent call last):
  File "/mnt/c/Users/Albi/documents/GitHub/hummingbot/hummingbot/core/utils/async_utils.py", line 7, in safe_wrapper
    return await c
TypeError: object NoneType can't be used in 'await' expression

I tried on linux mint and WSL, and the issue seems to appear on both, however, if I use the windows version (from installer) I don't get the issue

**Steps To Reproduce**
Here are the steps to reproduce the issue (see attachments in the section below):
1. Install the bot to linux from source
2. Turn paper_trading on

**Screenshots**
<img src="https://i.ibb.co/XpqYFVd/gh-1.png">

**Release version**
0.25.0

**Attachments**
(The issue persists in every strategy)
[logs_conf_pure_market_making_strategy_5.log](https://github.com/CoinAlpha/hummingbot/files/4408332/logs_conf_pure_market_making_strategy_5.log)
[conf_pure_market_making_strategy_5.txt](https://github.com/CoinAlpha/hummingbot/files/4408344/conf_pure_market_making_strategy_5.txt)

**Comment by rennel-tabing** [comment]
Error message:
`hummingbot.core.utils.async_utils - ERROR - Unhandled error in background task: object NoneType can't be used in 'await' expression
Traceback (most recent call last):`

I am able to reproduce this error message but it does not affect the bot from trading.

**Comment by albiDmtr** [comment]
Yes, the bot continues to trade, but I wasn't too comfortable seeing this message, and can't figure out the cause of it.

**Comment by dennisocana** [comment]
Installed from source on t2.xlarge AWS instance running on Ubuntu 18.04 LTS and did not get the same error. Will run some more tests in other environments.

<img width="960" alt="putty_R2hl1btmpc" src="https://user-images.githubusercontent.com/50150287/78100726-12b35080-7418-11ea-8828-2fc7c61cc5fa.png">

**Comment by znerol19619** [comment]
I was able to reproduce the error message in the log pane using a docker build 
![Screenshot from 2020-04-01 18-11-06](https://user-images.githubusercontent.com/27669951/78127087-22e22480-7446-11ea-8500-348e37c031f8.png)

**Version**
```
                "branch": "development",
                "commit": "f7b120d730e6e43e7996dccbe1254e119c020f17",
                "date": "2020-03-27T05:55:08Z",
```

**Comment by Clint-G13** [comment]
I was able to see this error yesterday while running a discovery strategy using the development branch.
![Capture](https://user-images.githubusercontent.com/57189990/78163504-10361280-747b-11ea-85c3-ef50394801c6.PNG)

```
Branch: Development
commit b421c3181589a27f6a938bd133f2d373dbad1a84
Date:  Wed Apr 1 10:45:37 2020 +0800
```

**Comment by Hashira-Lars** [comment]
Tested using WSL/Source/Local ubuntu 18.04 previously using latest dev `0.26.0` switch to master `0.25.0`

- Was able to reproduce behavior when your using Bamboo exchange on a strategy also causes strategy to not run on my end.

![image](https://user-images.githubusercontent.com/55869559/78204856-561bc680-74cd-11ea-82cb-cd617e696eb1.png)

**Comment by teotikalki** [comment]
I came here because I found this error after installing from source on a Debian 9 Proxmox VM. I'm also running it in paper trading mode with pure market making and binance. Latest master branch.

**Comment by dennisocana** [comment]
This is fixed by PR https://github.com/CoinAlpha/hummingbot/pull/1856 released in 0.28.0
